ocaml/typing/jkind.mli

```diff
@@ -417,6 +420,8 @@ val set_externality_upper_bound : t -> Externality.t -> t
 (*********************************)
 (* pretty printing *)
 
+val print_out_jkind : Format.formatter -> Outcometree.out_jkind -> unit
```

Comment:
I'm surprised to see this here, and not in oprint, which is where such functions normally live. Is there a reason to buck convention here? Same for the other new exported function here.

Reply:
At the time I had managed to convince myself that this was necessary to avoid a dependency cycle. But I think that was wrong. I'll move it.

Reply:
It's been moved

Reply:
But the `Outcometree.out_jkind` is still generated in the `Jkind` module. I'm not sure if that goes against convention? I wanted to keep the logic about printing jkinds in the `Jkind` module.

Reply:
I think probably generating the `out_jkind` here is OK. But I have a suspicion that there's a good reason for the module structure the way that it is. In any case, the processing done in the moved functions is a bit simple, and so I think it's OK for it to be elsewhere.